crates/driver/src/domain/quote.rs

```diff
@@ -103,7 +103,9 @@ impl Order {
                 kind: competition::order::Kind::Market,
                 app_data: Default::default(),
                 partial: competition::order::Partial::No,
-                interactions: Default::default(),
+                // TODO add actual pre- and post-interactions
```

Comment:
I'm not sure what this implies. Should we include any interactions in the quote? As always, I'd really appreciate opening an issue for the TODO with a detailed description of what needs to be done and why (and how, if it applies), unless this TODO is something really simple that you're going to do in a follow-up.

Reply:
An order with pre- or post-interactions should result in higher fees than the same order without these interactions as they required gas to run.
That means this somehow has to be factored into the quoting process. However, after bit more thought it's no longer clear to me who should be responsible for accounting for these gas costs. It could be the driver which would require us to pass the interactions as my comment suggests.
But it could also be centralized in the orderbook. The orderbook could estimate the gas costs of the pre- and post-interactions and just add that to the gas estimation of each quote it receives.

Reply:
So the user will specify the pre- and post- interactions when generating the quote?

Reply:
If the users are specifying interactions during quoting (they are, correct?) then this does indeed seem like the right place to implement this.

Reply:
https://github.com/cowprotocol/services/issues/1491